stripe-core/src/main/java/com/stripe/android/core/browser/customtabs/CustomTabsManager.kt

```diff
@@ -120,7 +120,7 @@ class CustomTabsManagerImpl @Inject constructor(
      *
      * @return a CustomTabsSession, or null if a connection to the service could not be established.
      */
-    private fun getSession(): CustomTabsSession? {
+    private fun getOrRetrieveSession(): CustomTabsSession? {
         session = client?.newSession(null)
```

Comment:
I think we’re now missing the `session = session ?: client?.newSession(null)` part to actually make it retrieve existing sessions.